bazel/envoy_build_system.bzl

```diff
@@ -456,7 +456,7 @@ def envoy_cc_test(
     if coverage:
         test_lib_tags.append("coverage_test_lib")
     envoy_cc_test_library(
-        name = name + "_lib",
+        name = name + "__lib",
```

Comment:
I'd suggest that we make this `_lib_internal_only`. That's clear and doesn't have the difficulty in visually distinguishing between one and two underscores :)

Reply:
Done. Went with `_lib_INTERNAL_ONLY`.